conf/application.conf

```diff
@@ -119,6 +119,11 @@ microservice {
       port = 9999
     }
 
+    identity-verification-frontend {
+      host = localhost
+      port = 9948
```

Comment:
This should be to 9938? 9948 is the stubs port, which IVF will redirect to 9938

Reply:
As discussed this points to the IV stub frontend 👍